### PR TITLE
Make wasm32 platform support docs easier to read

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ that not all uses of `wasm32-unknown-unknown` are using JavaScript and the web.
...
 ## Target maintainers
 
-When this target was added to the compiler platform-specific documentation here
+When this target was added to the compiler, platform-specific documentation here
 was not maintained at that time. This means that the list below is not
-exhaustive and there are more interested parties in this target. That being
-said since when this document was last updated those interested in maintaining
+exhaustive, and there are more interested parties in this target. That being
+said, when this document was last updated, those interested in maintaining
 this target are:
```
Comment (chenyukang) [inline src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md:36]: it seems it's harder to understand with `,` (I'm not a native English speaker)

Comment (tshepang) [reply src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md:36]: we can remove "when this document was last updated", such that we have

> That being said, those interested in maintaining this target...

Comment (chenyukang) [reply src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md:36]: yeah, make sense.

Comment (tshepang) [inline src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md:36]:
```suggestion
said, those interested in maintaining this target are:
```

File: src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md [hunk 2]
```diff
@@ -29,11 +29,10 @@ that not all uses of `wasm32-unknown-unknown` are using JavaScript and the web.
 
 ## Target maintainers
 
-When this target was added to the compiler platform-specific documentation here
+When this target was added to the compiler, platform-specific documentation here
 was not maintained at that time. This means that the list below is not
```
Comment (fbstj) [inline src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md:33]:
```suggestion
At the time this target was added to the compiler, platform-specific documentation here
was not maintained. This means that the list below is not
```
I belive this is a 'more easy to read' version?

Comment (tshepang) [reply src/doc/rustc/src/platform-support/wasm32-unknown-unknown.md:33]: yeah, that is an improvement

**Pull request merged**